labshare/migrations/0022_add_users_to_devices.py

```diff
@@ -1,28 +1,32 @@
 # Generated by Django 2.2.17 on 2020-12-10 09:36
+import binascii
+import os
 
-from django.contrib.auth.models import User
 from django.db import migrations
-from rest_framework.authtoken.models import Token
 
-from labshare.models import Device
+
+# Copied from rest_framework/authtoken/models.py.
+# This method is needed because token keys are not generated automatically in migrations.
+def generate_key():
+    return binascii.hexlify(os.urandom(20)).decode()
```

Comment:
I think it might be better to use the `secrets` [module](https://docs.python.org/3/library/secrets.html) here, since we ware creating passwords and need strong random numbers.

Reply:
This is literally the way how the Rest-Frameworks creates all the auth tokens. This is also not used for the password but only for the token.

Reply:
hmm, not the best idea of them, but then anyway. If they do that, we should follow!